Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,24 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    Dir.glob("#{formula.prefix}/**/*").each do |file|
+      path = Pathname.new file
+      next if path.directory?
+      basename = path.basename
+      return if !Metafiles.copy? basename.to_s
```

Comment:
This should end this one for all. https://github.com/xu-cheng/homebrew/commit/1dbdd31aa754383c8cdc73705e3eb4268a29e0f7